index.js

```diff
@@ -0,0 +1,192 @@
+const Manager = require("./lib/Manager");
+const Engineer = require("./lib/Engineer");
+const Intern = require("./lib/Intern");
+const inquirer = require("inquirer");
+const fs = require("fs");
+const generateTeam = require("./src/generateTeam");
+const { ensureExpectedIsNonNegativeInteger } = require("jest-matcher-utils");
+
+team = [];
+
+const managerQuestions = () => {
+  inquirer
+    .prompt([
+      {
+        type: "input",
+        name: "name",
+        message: "What is the team manager's name?",
+      },
+      {
+        type: "input",
+        name: "id",
+        message: "What is the team manager's ID?",
+      },
+      {
+        type: "input",
+        name: "email",
+        message: "What is the team manager's e-mail address?",
+      },
+      {
+        type: "input",
+        name: "officeNumber",
+        message: "What is the team manager's office number?",
+      },
+      {
+        type: "list",
+        name: "addTeamMember",
+        message: "Which team member/s would you like to add?",
+        choices: [
+          "Engineer",
+          "Intern",
+          "I don't want to add any more members.",
+        ],
+      },
+    ])
+    .then((managerAnswers) => {
+      const manager = new Manager(
+        managerAnswers.name,
+        managerAnswers.id,
+        managerAnswers.email,
+        managerAnswers.officeNumber
+      );
+      team.push(manager);
```

Comment:
It'd be great to have this logic in a seperate function - the more modular your code is, the more scalable it will be for the future. Try to think of each function as only having one job e.g.
one can ask the manager questions, another can be in charge of the loop where the user chooses what employee to add etc.